.github/workflows/ci.yml

```diff
@@ -7,10 +7,12 @@
 name: ci
 
 on:
-    push:
-    pull_request:
-        branches:
-            - main
+  push:
+    branches:
+    - main
+  pull_request:
+    branches:
+    - main
```

Comment:
This prevents the situation where GHA runs twice because the user didn't use a fork. Which is helpful since dependabot doesn't use forks.